api/graphql/schema.graphql

```diff
@@ -165,10 +162,189 @@ input UpdateUserInput {
     preferredLang: Language @rename(attribute: "preferred_lang")
 }
 
+input LocalizedStringInput {
+    en: String
+    fr: String
+}
+
+input CreateUserBelongsTo {
+    connect: ID
+}
+
+input CreatePoolBelongsTo {
+    connect: ID
+}
+
+input CreateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    level: Int!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input CreateClassificationBelongsToMany {
+    create: [CreateClassificationInput!]
+    connect: [ID!]
+}
+
+input CreateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateCmoAssetBelongsToMany {
+    create: [CreateCmoAssetInput!]
+    connect: [ID!]
+}
+
+input CreateOperationalRequirementInput {
+    key: String
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateOperationalRequirementBelongsToMany {
+    create: [CreateOperationalRequirementInput!]
+    connect: [ID!]
+}
+
+input CreatePoolCandidateInput {
+    pool: CreatePoolBelongsTo @belongsTo
+    user: CreateUserBelongsTo @belongsTo
```

Comment:
In this case, we want to allow for connect _or_ create on the user. But only connect on the pool, I think.